ftag/vds.py

```diff
@@ -52,11 +126,15 @@ def create_virtual_file(
     if not overwrite and out_fname.is_file():
         return out_fname
 
+    groups = list(h5py.File(fnames[0]).keys())
```

Comment:
It wasn't implemented before so understand if you don't want to fix, but I think it is probably worth checking here that all the files have compatible keys